whitenoise/compress.py

```diff
@@ -122,8 +126,22 @@ def write_data(self, path, data, suffix, stat_result):
         return filename
 
 
+try:
+    compressor_class = import_string(
+        getattr(
+            settings, "WHITENOISE_COMPRESSOR_CLASS", "whitenoise.compress.Compressor"
+        ),
+    )
+except ImproperlyConfigured:
+    compressor_class = Compressor
```

Comment:
reading settings at import time is not generally safe in Django, since they can still be overriden, in e.g. tests or by later configuration settings loading. Only read the setting at runtime

Reply:
This should be fixed.